src/staticanalysis/bot/static_analysis_bot/clang/tidy.py

```diff
@@ -244,8 +244,13 @@ def __init__(self, header_data, revision):
         self.body = None
         self.notes = []
 
+        self.reason = None
+        for c in settings.clang_checkers:
+            if c['name'] == self.check:
+                self.reason = c['reason']
+
     def __str__(self):
-        return '[{}] {} {} {}:{}'.format(self.type, self.check, self.path, self.line, self.char)
+        return '[{}] {} {} {} {}:{}'.format(self.type, self.check, self.reason, self.path, self.line, self.char)
```

Comment:
Nit: Since we don't include `self.body` here, let's also not include `self.reason`, in order to keep the output short.